src/controllers/url.ts

```diff
@@ -0,0 +1,86 @@
+import { Request, Response } from 'express';
+import { customAlphabet } from 'nanoid';
+import Url from '../models/url';
+import { BASE_URL, HTTP_ERRORS } from '../constant';
+import logger from '../logger/index';
+
+const createShortUrls = async (req: Request, res: Response) => {
+  try {
+    const { url, customName } = req.body;
+    let shortUrls;
```

Comment:
I don't see the point of declaring this here. You can just use `const` on line 34.

Reply:
If I don't define shortUrls before the if else statement and use const, I get this error: Cannot find name 'shortUrls'. Which was why I defined it in the first place.